Shared/Samples/Augment reality to navigate route/AugmentRealityToNavigateRouteView.swift

```diff
@@ -0,0 +1,253 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import AVFoundation
+import SwiftUI
+
+struct AugmentRealityToNavigateRouteView: View {
+    /// The data model for the selected route.
+    @StateObject private var routeDataModel = RouteDataModel()
+    /// A Boolean value indicating whether the route planner is showing.
+    @State private var isShowingRoutePlanner = true
+    /// The location datasource that is used to access the device location.
+    @State private var locationDataSource = SystemLocationDataSource()
+    /// A scene with an imagery basemap.
+    @State private var scene = Scene(basemapStyle: .arcGISImagery)
+    /// The elevation surface set to the base surface of the scene.
+    @State private var elevationSurface: Surface = {
+        let elevationSurface = Surface()
+        elevationSurface.navigationConstraint = .unconstrained
+        elevationSurface.opacity = 0.5
+        elevationSurface.backgroundGrid.isVisible = false
+        return elevationSurface
+    }()
+    /// The elevation source with elevation service URL.
+    @State private var elevationSource = ArcGISTiledElevationSource(url: URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!)
+    /// The graphics overlay containing a graphic.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    /// The status text displayed to the user.
+    @State private var statusText = "Adjust calibration before starting."
+    /// A Boolean value indicating whether the use is navigatig the route.
+    @State private var isNavigating = false
+    /// The result of the route selected in the route planner view.
+    @State private var routeResult: RouteResult?
+    
+    init() {
+        elevationSurface.addElevationSource(elevationSource)
+        scene.baseSurface = elevationSurface
+    }
+    
+    var body: some View {
+        if isShowingRoutePlanner {
+            RoutePlannerView(isShowing: $isShowingRoutePlanner)
+                .onDidSelectRoute { routeGraphic, routeResult  in
+                    self.routeResult = routeResult
+                    graphicsOverlay = makeRouteOverlay(
+                        routeResult: routeResult,
+                        routeGraphic: routeGraphic
+                    )
+                }
+                .task {
+                    Task { try await elevationSource.load() }
```

Comment:
`Task` shouldn't be used just to discard an error.

```suggestion
                    try? await elevationSource.load()
```